department-of-veterans-affairs/va.gov-cms
# VACMS-2810: Date field audit view.

## Description

See  #2810. 

## Testing done

Manual, behat.


## Screenshots
![Content_with_date_fields___VA_gov_CMS](https://user-images.githubusercontent.com/643678/95083150-1935c400-06ea-11eb-8e06-c2ce5c6c5edf.jpg)


## QA steps

As a content admin or admin, i can go to /admin/content/audit/content-with-date-fields and see all existing date fields data. 

This view could be used in the future to QA the migration to a time zone based field.
